odra-casper/livenet-env/src/livenet_host.rs

```diff
@@ -209,3 +218,21 @@ impl HostContext for LivenetHost {
         rt.block_on(async { client.transfer(to, amount, timestamp).await })
     }
 }
+
+fn find_wasm_file_path(wasm_file_name: &str) -> PathBuf {
```

Comment:
This lived in `casper_client.rs`, have you moved it her, or copied?

Reply:
Copied, now it is removed